logging/slf4j/src/main/java/io/helidon/logging/slf4j/Slf4jMdcPropagator.java

```diff
@@ -29,7 +31,7 @@
 
     @Override
     public Map<String, String> data() {
-        return MDC.getCopyOfContextMap();
+        return Optional.ofNullable(MDC.getCopyOfContextMap()).orElseGet(HashMap::new);
```

Comment:
If mutable map is not required, please use `Map::of`

Reply:
changed